scripts/copy-files.ts

```diff
@@ -1,8 +1,10 @@
 import * as fs from "node:fs/promises";
 import * as path from "node:path";
+import * as glob from "glob";
 
 const cp = async (fromGlob: string, toPath: string) => {
-    for await (const file of fs.glob(fromGlob)) {
+    const files = glob.sync(fromGlob);
```

Comment:
nit: Now that I look at it... this is `async` function that does `glob.sync`, and then we have `for await`. Not that it will change anything, given there's only a few files, but I'd rather make it async :)

Reply:
There's only a callback API there as far as I can tell, so I just changed `for await` to `for`